src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
-package core.basesyntax;
+package org.example;
+
+import java.util.Random;
 
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    //Creating type of color
+    public static Color getRandomColor() {
+
+        Random random = new Random();
```

Comment:
random should be instantiated once and reused multiple times, 

nor creating each time the getRandomColor method is invoked